examples/connector/java/src/main/java/connector/ConnectorServiceImpl.java

```diff
@@ -81,101 +83,73 @@ public void schema(SchemaRequest request, StreamObserver<SchemaResponse> respons
     @Override
     public void update(UpdateRequest request, StreamObserver<UpdateResponse> responseObserver) {
         Map<String, String> configuration = request.getConfigurationMap();
-        String state_json = request.hasStateJson() ? request.getStateJson() : "{}";
+        String stateJson = request.hasStateJson() ? request.getStateJson() : "{}";
         Selection selection = request.hasSelection() ? request.getSelection() : null;
 
         ObjectMapper mapper = new ObjectMapper();
         UpdateResponse.Builder responseBuilder = UpdateResponse.newBuilder();
 
         try {
-            State state = mapper.readValue(state_json, State.class);
+            State state = mapper.readValue(stateJson, State.class);
 
             // -- Send a log message
-            responseBuilder.clear();
-            responseObserver.onNext(responseBuilder
-                    .setLogEntry(LogEntry.newBuilder()
-                            .setLevel(LogLevel.INFO)
-                            .setMessage("Sync STARTING")
-                            .build())
-                    .build());
+            System.out.println("[Update]: Sync STARTING");
```

Comment:
log should be in json format - https://github.com/fivetran/fivetran_sdk/blob/main/development-guide.md#logging

Reply:
The log should be in json format. 

`{"level": "INFO", "message": "Sync Starting"}`